starcheck/utils.py

```diff
@@ -13,6 +13,8 @@
 from astropy.table import Table
 from Chandra.Time import DateTime
 from chandra_aca.star_probs import guide_count, mag_for_p_acq
+from chandra_aca.dark_model import dark_temp_scale
+from sparkles import get_t_ccds_bonus
```

Comment:
As a continuing evolution of best practices (and something @javierggt already does a lot)... In a case like this where the sparkles `get_t_ccds_bonus` function is a bit ambigous with the local `_t_ccds_bonus`, the preferred use would be:
```
import sparkles
...
blah = sparkles.get_t_ccds_bonus(...)
```